fastplotlib/layouts/_plot_area.py

```diff
@@ -214,7 +228,7 @@ def controller(self, new_controller: str | pygfx.Controller):
         # TODO: monkeypatch until we figure out a better
         #  pygfx plans on refactoring viewports anyways
         if self.parent is not None:
-            if self.parent.__class__.__name__ == "Figure":
+            if self.parent.__class__.__name__.endswith("Figure"):
```

Comment:
our CI caught this bug in the tests that checks to make sure controllers can be changed, I didn't notice :D